Xamarin.Forms.Core/PlatformConfiguration/AndroidSpecific/TabsPlacement.cs

```diff
@@ -0,0 +1,17 @@
+namespace Xamarin.Forms.PlatformConfiguration.AndroidSpecific
+{
+	public enum TabsPlacement
+	{
+		/// <summary>
```

Comment:
I copy/pasted from the WindowsSpecific\ToolbarPlacement.cs file. Anyway, should I just delete the comments? Also, I couldn't find the mdoc files :(

Reply:
@mikescandy  yea just delete the comments.. Since you rebased with master you won't need to run the docs file that generates docs anymore. Just make sure the API Changes part of the PR has everything